src/bin/cli/rewards/community_advisors.rs

```diff
@@ -84,6 +90,10 @@ impl CommunityAdvisors {
             &approved_proposals,
             &fund_settings.into(),
             &rewards_slots.into(),
+            Seed::from(
+                Digest::<Blake2b256>::from_str(&seed)
```

Comment:
This does not hash the input string, but rather tries to interpret it as a black2b256 digest. I think we should do this the other way around